doc/features.rst

```diff
@@ -17,7 +17,7 @@ ________
   
 * Supports a truly reproducible workflow by minimizing the code needed for analysis and visualization.
 * Already used in a variety of research projects, from conception to final publication.
-* All HoloViews objects can be pickled and unpickled.
+* All HoloViews objects can be pickled and unpickled, with no plotting-library dependencies.
```

Comment:
Here I was referring to the objects, not the styles.  If we just use unpickle, we'll only get the objects, right?  And thus won't have to worry about the plotting library dependencies?  The point is to make it clear that the data will always, always be able to be extracted (as long as pickles are supported in general), and thus that it's a safe way to store the data.